clio-client/src/main/scala/org/broadinstitute/clio/client/parser/BaseArgs.scala

```diff
@@ -104,6 +105,36 @@ class BaseParser extends scopt.OptionParser[BaseArgs]("clio-client") {
         .text("The project for this whole genome unmapped bam.")
     )
 
+  cmd(Commands.MoveWgsUbam.toString)
+    .action((_, c) => c.copy(command = Some(Commands.MoveWgsUbam)))
+    .text(
+      "This command is used to copy a unmapped bam from one location to another. Only implemented for cloud unmapped bams"
```

Comment:
I can't find any code that asserts the location is only `Location.GCP`. We should be checking it, and raising a nice error if something else is given.

Reply:
Fixed.